app/src/main/java/com/main/mealplanner/MainViewModel.kt

```diff
@@ -10,8 +10,8 @@ class MainViewModel : ViewModel() {
     var recipes: MutableLiveData<ArrayList<RecipeHeader>> = MutableLiveData()
     var recipeService: RecipeService = RecipeService()
 
-    fun fetchRecipe(s: String) {
-        recipes = recipeService.fetchRecipe(s)
+    fun fetchRecipe(meal: String) {
+        recipes = recipeService.fetchRecipe(meal)
```

Comment:
Renamed the variable so it becomes easier to understand.
Source:
https://kotlinlang.org/docs/coding-conventions.html#function-names

Reply:
Good.  This came up in a previous review as well...